client/src/Pages/StudentUpcomingPage.js

```diff
@@ -0,0 +1,188 @@
+import React from "react";
+import profile from '../Images/profile.jpg'; 
+import {Card, CardDeck, Button} from "react-bootstrap";
+import google from '../Images/google.jpg'; 
+import microsoft from '../Images/microsoft.jpg'; 
+import facebook from '../Images/facebook.jpg'; 
+import apple from '../Images/apple.jpg'; 
+import tesla from '../Images/tesla.jpg'; 
+import snapchat from '../Images/snapchat.jpg'; 
+import qualcomm from '../Images/qualcomm.jpg'; 
+import paypal from '../Images/paypal.jpg'; 
+import netflix from '../Images/netflix.jpg'; 
+import {MoreInfo} from './MoreInfo'
+
+export default class StudentUpcomingPage extends React.Component {
+    handleRoute = route => () => {
+        this.props.history.push({ pathname: route });
+        };
+  render() {
+    return (
+        <div style={{ "text-align": "center", "margin": "20px 20px" }}>
+            <Card style={{"box-shadow": "0 4px 8px 0 rgba(0,0,0,0.2)", "margin-bottom": "20px"}}>
+            <br></br>
+            <h2><b>Upcoming: </b><b>Career Fair #2</b></h2>
+            <br></br>
+            <h5><b>Status: </b>Registered</h5>
+            <br></br>
+            <h5><b>Date: </b>Monday, October 26, 2020</h5>
+            <br></br>
+            <h5><b>Time: </b>11AM PST - 5PM PST</h5>
+            <br></br>
+            </Card>
+            <CardDeck>
+            <Card style={{"box-shadow": "0 4px 8px 0 rgba(0,0,0,0.2)"}}>
+                <div style={{ "width": "200px", "margin": "auto"}}> 
+                <Card.Img variant="top" src={google} height="200" />
+                </div>
+                <Card.Body>
+                <Card.Title><b>Google</b></Card.Title>
+                <Card.Text>
+                <p><b>Role: </b>Seeking software engineer interns.</p>
+                <p></p>
+                <p><b>Year: </b>Sophomores and Juniors. This internship is intended for students who are pursuing a Bachelor's degree program in Computer Science or a related field with an anticipated graduation date after December 2021, depending on their program and unique circumstances.</p>
+                </Card.Text>
+                </Card.Body>
+                <Card.Footer>
+                <MoreInfo></MoreInfo>
+                </Card.Footer>
+            </Card>
```

Comment:
This is good, but to make integration with backend easier later, could we create this dynamically. 
For now it would be as simple as:
- replacing career fair name with this.props.careerFair
-  adding a list of 'company' objects (pre-populated as you have here) and having a for loop render them. 
So that when we integrate with the backend, it's just a matter of using this.props.careerFair to get the other career fair details like date & time, and to query the API and set the list of company objects correctly.

Reply:
for sure, i will work on creating components dynamically once i get all the basic layouts for these pages done!